pkg/eventprocessor/eventfeed/impl/eventfeed.go

```diff
@@ -218,6 +222,28 @@ func (ef *EventFeed) Start(
 	return nil
 }
 
+// removeDuplicateLogs removes duplicate logs from the list of logs
+// This is needed because some node RPC endpoints can return duplicate logs
+// for a given block range. This is a known bug in FVM and impacts Filecoin
+// and Filecoin Hyperspace nodes.
+// See: https://github.com/filecoin-project/ref-fvm/issues/1350
+func (ef *EventFeed) removeDuplicateLogs(logs []types.Log) []types.Log {
+	seenLogs := make(map[string]bool)
+	var uniqueLogs []types.Log
+	for l := range logs {
+		uniqueLogID := fmt.Sprintf("%d%s%d", logs[l].BlockNumber, logs[l].TxHash.String(), logs[l].Index)
+
+		// skip duplicate logs
+		if _, ok := seenLogs[uniqueLogID]; ok {
+			continue
```

Comment:
i think it's worth adding a log here to watch how much this is happening

Reply:
yes, good idea